docs/diagrams/PieChartWindowClassDiagram.puml

```diff
@@ -0,0 +1,47 @@
+@startuml
+skinparam backgroundColor #ffffff
+skinparam classAttributeIconSize 0
+hide circle
+
+abstract class "{abstract}\nUiPart" {
+    + {abstract} getRoot()
+}
+
+class SummariseCommand {
+    /' Fields '/
+    -{static}positiveStatsByFacultyData : TreeMap
+    -{static}covidStatsByBlockDataList : TreeMap
+
+    /' Methods '/
+    + getCovidStatsByBlockDataList()
+    + getPositiveStatsByFacultyData()
+    + getHighestPositiveByFaculty()
+}
+
+class PieChartWindow {
+    /' Fields '/
+    - {static} logger : Logger
+    - {static} FXML : String
+    - covidStatsByBlockData : TreeMap
+    - positiveStatsByFacultyData : TreeMap
+    - pieChart : PieChart
+    - charts : VBox
+    - chartScene : Scene
+
+    /' Methods '/
+    + PieChartWindow(root : Stage)
```

Comment:
I think you can omit the variable name of the parameter for root
```suggestion
    + PieChartWindow(root)
```

Reply:
Thanks for the suggestions. I have made the amendments.